src/views/Project/index.jsx

```diff
@@ -0,0 +1,37 @@
+import { Component } from 'react';
+import Typography from 'material-ui/Typography';
+import ReactMarkdown from 'react-markdown';
+import projects from '../../data/loader';
+import BugsTable from '../../components/BugsTable';
+
+class Project extends Component {
```

Comment:
You can do `export default class Project extends Component {` and remove the default export at the bottom.